### PR TITLE
[acrtransfer] acrtransfer extension v1.1 release

### PR DESCRIPTION
**Pull request by mabenedi**
Change log:

- Add: New command acr pipeline-run clean - Bulk deletes failed pipeline-runs.
- Add: New parameter to acr pipeline-run list --top - Displays only the n most recent pipeline-runs.
- Bug fix:  ResourceNotFoundError will now be thrown if the resource does not exist when calling import-pipeline delete and export-pipeline delete.
- Bug fix: Help text typos fixed.
- Style change: Changed from distutils logger to knock.log logger.
- Style change: General style fixes to conform to latest azdev style output.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required) **Yes**
- [x] Have you run `python scripts/ci/test_index.py -q` locally? **Yes**

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
acrtransfer

**Review by wangzelin007 — CHANGES_REQUESTED** [review]
Please write the description of changes into HISTORY.rst

File: src/acrtransfer/azext_acrtransfer/_params.py [hunk 1]
```diff
@@ -35,3 +35,7 @@ def load_arguments(self, _):
         c.argument('force_update_tag', options_list=['--force-redeploy', '-f'], action='store_true', help='Flag that forces ARM to redeploy resource even if no parameters have changed.')
         c.argument('storage_blob_name', options_list=['--storage-blob', '-b'], help='Name of the storage blob to be imported or exported.')
         c.argument('artifacts', options_list=['--artifacts', '-a'], nargs='+', help='Space-separated list of container artifacts. Artifacts must be either of the form `repository:tag` or `repository@digest` such as hello-world:latest or hello-world@sha256:90659bf80b44ce6be8234e6ff90a1ac34acbeb826903b02cfa0da11c82cbc042.')
+        c.argument('top', options_list=['--top'], validator=validate_top, help='List only the last n pipeline-runs. Must be an integer.')
+
```
Comment (wangzelin007) [inline src/acrtransfer/azext_acrtransfer/_params.py:39]: Only used when a option_list is needed, CLI will handle the default.
Default rule is mapping `argument_name` to `--argument-name`.

Comment (mabenedi) [reply src/acrtransfer/azext_acrtransfer/_params.py:39]: fixed

**Comment by mabenedi** [comment]
> Please write the description of changes into HISTORY.rst

Added description of changes into HISTORY.rst

**Review by wangzelin007 — APPROVED** [review]

File: src/acrtransfer/azext_acrtransfer/_params.py [hunk 2]
```diff
@@ -35,3 +35,7 @@ def load_arguments(self, _):
...
+    with self.argument_context('acr pipeline-run clean') as c:
+        c.argument('dry_run', options_list=['--dry-run'], action='store_true', help='List the failed pipeline-runs that would have been deleted, but do not delete any.')
```
[the remark this answers is not in the document]
Comment (mabenedi) [reply src/acrtransfer/azext_acrtransfer/_params.py]: fixed

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**